.github/workflows/ci.yml

```diff
@@ -188,11 +188,11 @@ jobs:
       earthly_runner_address: ${{ secrets.earthly_runner_address }}
       earthly_runner_secret: ${{ secrets.earthly_runner_secret }}
   deploy:
-    uses: input-output-hk/catalyst-ci/.github/workflows/deploy.yml@master
+    uses: input-output-hk/catalyst-ci/.github/workflows/deploy.yml@fdcDeprecation
```

Comment:
A reminder: Don't forget to change the branches `@fdcDeprecation` to `@master` once everthing is done and ready to merge

Reply:
I searched for the @fdcDeprecation branches in my PR already but couldn't find any for now deeming it's been taken care of

Reply:
I mean the reason why we currently using `@fdcDeprecation` is just for testing the functionality that is implemented in the branch. So when everything works perfectly, we need to change from `@fdcDeprecation` back to `@master` before merging this branch into master.

Reply:
I believe I changed back to @master for the whole repo already fingers crossed